src/Microsoft.AspNetCore.Http/Microsoft.AspNetCore.Http.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>ASP.NET Core default HTTP feature implementations.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netcoreapp2.2</TargetFrameworks>
```

Comment:
2.2 is so retro tho

Reply:
3.0 is the best yet; though hard to reference, at this time 😢